File: CatalogAPI/GetCatalogForPoint.py
import requests

lat = 55.862261
lng = -4.250959
zoom = 16
token = "ADD YOUR TOKEN HERE"

url = ("https://catalog-api.hibirdi.com/catalog/lat/{}/lng/{}/zoom/{}/token/{}"
       .format(lat, lng, zoom, token))

response = requests.request("GET", url)

print(response.text)

File: CatalogAPI/GetCatalogForPoint.js
let data = null;
const xhr = new XMLHttpRequest();

const lat = 55.862261;
const lng = -4.250959;
const zoom = 16;
const token = "ADD YOUR TOKEN HERE"
const url = "https://catalog-api.hibirdi.com/catalog/lat/" + lat +
            "/lng/" + lng +
            "/zoom/" + zoom +
            "/token/" + token;

xhr.addEventListener("readystatechange", function () {
  if (this.readyState === 4) {
    console.log(this.responseText);
  }
});

xhr.open("GET", url);
xhr.send(data);
